### ISSUE TITLE
Ensure AdSense widgets only show "Gathering data" box when API is actually gathering data

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#4087]
Currently, Site Kit widgets are configured to show the "AdSense gathering data" box whenever the API returns a response with all 0s. This is somewhat confusing and an incorrect approach, since it could also mean that the site actually has 0 data, even though data is already being aggregated properly.

This should be adjusted, and the "AdSense gathering data" box should only be shown for 0 data if the API is actually _likely_ gathering data - _likely_ since this needs to be based on heuristics as there is no API flag that clearly states whether or not data is now ready to be served.

Related: #4085, #4086

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by felixarntz** [comment]
@kuasha420 Based on the changed requirements in #4085 and #4086, which also affect this one, we'll have to hold off here, I'll move this back to the Acceptance Criteria column.

@eclarke1 @fhollis Based on those changes in the other two issues, we actually have to re-evaluate our options for this one here, so it won't be feasible to work on it this sprint, and it isn't as high of a priority as the other two anyway. I'll remove it from the current sprint, we can pick this up again once we have a path forward.

**Comment by kuasha420** [comment]
@felixarntz yes, that's perfect. I figured as much and notified @fhollis about it earlier today. :)

**Comment by eclarke1** [comment]
@felixarntz did we have a path forward for this one yet at all? Wondering if it's something we want to get into Sprint 60 as it will help the Zero Data State epic as a whole? cc @fhollis

**Comment by felixarntz** [comment]
@eclarke1 This one is a lower priority for now - we still need to figure out how to handle this. But it shouldn't block the zero states work.

**Comment by eclarke1** [comment]
@felixarntz just wanted to revisit this ticket now we have design approval on Zero Data States and are progressing that epic soon. I know you mentioned this was not blocking it, but potentially feels like something we could do alongside, and therefore we may want to revisit this one soon? WDYT?
cc @fhollis

**Comment by felixarntz** [comment]
@eclarke1 This is kind of unrelated to the UX work, it's more of a technical thing we should improve at some point, but it's not as high a priority. The zero data state UX work doesn't block this and vice-versa.

With that said, I'm planning to pick up research on this again soon.
